README.md

```diff
@@ -31,13 +32,8 @@ I've hit a bit of a design roadblock for one of the bigger features that are in
 <img src="https://raw.githubusercontent.com/elkowar/dots-of-war/master/eww-bar/.config/eww-bar/showcase.png" height="400" width="auto"/>
 
 * [Vertical Bar by Rxyhn](https://github.com/rxyhn/bspdots)
-
-<div align="left">
-
 ![Rxyhn-rice](https://user-images.githubusercontent.com/93292023/152228869-d618335a-7a1e-40f7-95f9-b1cf401be89e.gif)
 
-</div>
```

Comment:
i missed that earlier:
this div is needed, otherwise things look weird

Reply:
Resolved by: https://github.com/elkowar/eww/pull/886#issuecomment-2310553020